homeassistant/components/honeywell/config_flow.py

```diff
@@ -79,7 +89,7 @@ async def async_step_reauth_confirm(
 
         return self.async_show_form(
             step_id="reauth_confirm",
-            data_schema=REAUTH_SCHEMA,
+            data_schema=reauth_schema,
```

Comment:
```suggestion
            data_schema=self.add_suggested_values_to_schema(REAUTH_SCHEMA, self.entry.data),
```

Reply:
That make sense... I must have stopped reading the docs at the schema default section :)